packages/lib/src/artifacts/ZosContract.ts

```diff
@@ -0,0 +1,111 @@
+import BN from 'bignumber.js';
+import ZWeb3 from './ZWeb3';
+import { getSolidityLibNames, hasUnlinkedVariables } from '../utils/Bytecode';
+import { Contract, TransactionObject, BlockType } from 'web3-eth-contract';
+import { Callback, EventLog, EventEmitter, TransactionReceipt } from 'web3/types';
+import Contracts from './Contracts';
+import { StorageLayoutInfo } from '../validations/Storage';
+import _ from 'lodash';
+
+/*
+ * ZosContract is an interface that extends Web3's Contract interface, adding some properties and methods like:
+ * address getter: retrieves the deployed address
+ * schema: compilation artifacts
+ * new(): deployes a new contract
+ * at(): retrieves a deployed contract at the specified address
+ * link(): links libraries in a contract schema
+ */
+export default interface ZosContract {
```

Comment:
Is it easy with the IDE you are using to rename ZosContract to plainly Contract?

Reply:
Yes, let's rename. I'll merge anyway now so we can start testing though, but it'd be good to have another PR renaming it.